### ISSUE TITLE
[BUG] configmap flag still in all docs but not available anymore, unable to use with benchmark-operator now

### ISSUE THREAD

**Issue by MansM** [kube-burner/kube-burner#648]
# Bug Description

## **Output of `kube-burner` version**
```
Version: v1.9.6
Git Commit: a51ce29e53ac6e4f022818e642295d993ae695f5
Build Date: 2024-05-16-13:49:35
Go Version: go1.21.10
OS/Arch: linux amd64
```

## **Describe the bug**

Kubeburner docs still list --configmap as a valid option for providing config
benchmark-operator leverages this to run the benchmark inside a cluster
this commit removes the feature: https://github.com/kube-burner/kube-burner/pull/611/commits/2ff7db3898182b3836c254726736aaa37a6f54a6

Docs still listing it as available
https://kube-burner.github.io/kube-burner/v1.9.6/cli/#init
https://github.com/kube-burner/kube-burner/blob/main/docs/cli.md

## **To Reproduce**

1. kube-burner --configmap....

## **Expected behavior**

kube-burner retrieving config from configmap (as in 1.9.3)
## **Screenshots or output**

If applicable, add screenshots or kube-burner output to help explain your problem.

**Comment by vishnuchalla** [comment]
cc: @rsevilla87 ^^

**Comment by MansM** [comment]
Not exactly the solution I hoped for.

**Comment by vishnuchalla** [comment]
@MansM looks like it got closed automatically as soon as the PR got merged. Reopening the issue.

**Comment by MansM** [comment]
Yes. That would be awesome. Sent from my iPhone Op 24 sep 2024 om 05:39 heeft Supraja Manda ***@***.***> het volgende geschreven:﻿
Hi @MansM, I'm interested in working on this issue. So the expectation is to get back the functionality of --configmap in kube-burner?
